SerialPrograms/Source/PokemonSwSh/MaxLair/Inference/PokemonSwSh_MaxLair_Detect_PathSide.cpp

```diff
@@ -5,6 +5,7 @@
  */
 
 #include <cmath>
+#include "Kernels/BinaryMatrix/Kernels_SparseBinaryMatrixBase.tpp"

```

Comment:
Something is wrong here. The entire class is explicitly instantiated in `Kernels_BinaryMatrix.cpp`.

What's the error message you're getting?

Reply:
I guess it is the same that the warning I got there 
https://github.com/pifopi/Arduino-Source/runs/4914462996?check_suite_focus=true#step:8:370